nym-wallet/src/components/Delegation/DelegateModal.tsx

```diff
@@ -184,23 +184,24 @@ export const DelegateModal: React.FC<{
         }
       }}
       header={header || 'Delegate'}
-      subHeader="Delegate to mixnode"
       okLabel={buttonText || 'Delegate stake'}
       okDisabled={!isValidated}
       sx={sx}
       backdropProps={backdropProps}
     >
-      <IdentityKeyFormField
-        required
-        fullWidth
-        placeholder="Node identity key"
-        onChanged={handleIdentityKeyChanged}
-        initialValue={identityKey}
-        readOnly={Boolean(initialIdentityKey)}
-        textFieldProps={{
-          autoFocus: !initialIdentityKey,
-        }}
-      />
+      <Box sx={{ mt: 1.8 }}>
```

Comment:
Can we keep these to whole numbers. Make this `2`. In some cases we can use halves e.g `0.5, 1.5 3.5 etc`.
Otherwise we'll lose consistency in our spacing.